src/components/TodoModal/TodoModal.tsx

```
@@ -1,12 +1,35 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+interface TodoModalProps {
+  userId: number;
+  todo: Todo | null;
+  closeModalWindow: (v: Todo | null) => void;
+}
+
+export const TodoModal: React.FC<TodoModalProps> = ({
+  userId,
+  todo,
+  closeModalWindow = () => {},
```

Issue: You are providing a default value for `closeModalWindow` in the destructuring assignment. Since this prop is required in the `TodoModalProps` interface and always provided by the parent, the default value is unnecessary and could hide missing prop issues. Remove the default value to ensure prop requirements are enforced.